astrosat_users/serializers/serializers_customers.py

```diff
@@ -13,10 +13,11 @@
 class CustomerSerializer(serializers.ModelSerializer):
     class Meta:
         model = Customer
-        fields = ("id", "type", "name", "title", "description", "logo", "url", "country", "address", "postcode")
+        fields = ("id", "type", "name", "official_name", "comp_type", "registered_id","description", "logo", "url", "country", "address", "postcode")
 
     id = serializers.UUIDField(read_only=True)
     type = serializers.CharField(source="customer_type")
+    comp_type = serializers.CharField(source="company_type")
```

Comment:
Why did you call the field `comp_type` instead of `company_type`?

Reply:
I got a clash if I remember correctly as the source was already `company_type`. I will however double check that, more likely it was something else and I assumed it was a clash. I would prefer it also, to be `company_type`.

Reply:
If the names are the same, then you don't need the `source` argument; that's only there if you want to use a different name for the serializer field than the model field.

Reply:
You are right of course, I have removed this line entirely, as well as the accompanying line in the test, you mention below. I'm about to push this change.